.github/workflows/build-keycloak-container-image.yml

```diff
@@ -43,17 +43,16 @@ jobs:
         run: scripts/build.sh
         env:
           DOCKER_REGISTRY: ${{ secrets.DOCKER_REGISTRY }}
-          IMAGE: keycloak
-          REPOSITORY: scs/keycloak
+          IMAGE: scs-keycloak
+          REPOSITORY: scs-container-images/scs-keycloak
           VERSION: ${{ matrix.version }}
 
       - name: Push container image
-        run: |
-          scripts/push.sh
+        run: scripts/push.sh
         env:
           DOCKER_REGISTRY: ${{ secrets.DOCKER_REGISTRY }}
-          IMAGE: keycloak
-          REPOSITORY: scs/keycloak
+          IMAGE: scs-keycloak
+          REPOSITORY: scs-container-images/scs-keycloak
```

Comment:
The fact that it is an SCS image results from the fact that the image is in an SCS namespace? I would prefer to keep keycloak as the name. I think it is better to add a label if this is to be listed explicitly.

Reply:
As discussed on the weekly IAM meeting of kw3, nobody veto the `scs-keycloak` name, so we keep it to differentiate it from a base Keycloak image as we have plugings already instaled.